### PR TITLE
AAP-21516 Update links to use BaseURL attribute in platform modules

### PR DESCRIPTION
**Pull request by ariordan-redhat**
Update the platform modules to use the `{BaseURL}` attribute for links.

This replaces `https://access.redhat.com/documentation/en-us` with `{BaseURL}`

### CODE REVIEW

File: downstream/modules/platform/con-aap-installation-on-disconnected-rhel.adoc [hunk 1]
```diff
@@ -24,4 +24,4 @@ The RHEL Binary DVD method requires the DVD for supported versions of RHEL, incl
 ====
 
 .Additional Resources
-* link:https://access.redhat.com/documentation/en-us/red_hat_satellite/6.11/html/installing_satellite_server_in_a_disconnected_network_environment/index[Satellite]
\ No newline at end of file
+* link:{BaseURL}/red_hat_satellite/6.11/html/installing_satellite_server_in_a_disconnected_network_environment/index[Satellite]
```
Comment (ianf77) [inline downstream/modules/platform/con-aap-installation-on-disconnected-rhel.adoc:27]: It's satellite 6.14 now.

File: downstream/modules/platform/con-about-eda-controller.adoc [hunk 2]
```diff
@@ -16,5 +16,5 @@ The {EDAcontroller} is the interface for event-driven automation and introduces
 The following link will not work until published.
 ////
 
-* link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/getting_started_with_event-driven_ansible_guide/index[Getting Started with Event-Driven Ansible Guide].
+* link:{BaseURL}/red_hat_ansible_automation_platform/2.4/html/getting_started_with_event-driven_ansible_guide/index[Getting Started with Event-Driven Ansible Guide].
```
Comment (ianf77) [inline downstream/modules/platform/con-about-eda-controller.adoc:19]: {PlatformVers} ?

File: downstream/modules/platform/con-building-an-execution-environment-in-a-disconnected-environment.adoc [hunk 3]
```diff
@@ -6,7 +6,7 @@
 = Building an {ExecEnvShort} in a disconnected environment
 
 
-link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/creating_and_consuming_execution_environments/index[Creating execution environments] for {PlatformNameShort} is a common task which works differently in disconnected environments. When building a custom {ExecEnvShort}, the ansible-builder tool defaults to downloading content from the following locations on the internet:
+link:{BaseURL}/red_hat_ansible_automation_platform/2.4/html/creating_and_consuming_execution_environments/index[Creating execution environments] for {PlatformNameShort} is a common task which works differently in disconnected environments. When building a custom {ExecEnvShort}, the ansible-builder tool defaults to downloading content from the following locations on the internet:
```
Comment (ianf77) [inline downstream/modules/platform/con-building-an-execution-environment-in-a-disconnected-environment.adoc:9]: {PlatformVers}

File: downstream/modules/platform/proc-installing-the-aap-setup-bundle.adoc [hunk 4]
```diff
@@ -1,4 +1,4 @@
-// this info is duplicated here: https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/1.2/html/installing_and_upgrading_private_automation_hub/installing_online_or_offline#doc-wrapper
+// this info is duplicated here: {BaseURL}/red_hat_ansible_automation_platform/1.2/html/installing_and_upgrading_private_automation_hub/installing_online_or_offline#doc-wrapper
```
Comment (ianf77) [inline downstream/modules/platform/proc-installing-the-aap-setup-bundle.adoc:1]: Is this the link that's actually included? To 1.2?

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 5]
```diff
@@ -26,7 +26,7 @@ When you configure your {PrivateHubName} with LDAP authentication, you must set
 
 .Procedure
 
-. Access your inventory file according to the procedure in link:hhttps://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html-single/red_hat_ansible_automation_platform_installation_guide/index#proc-editing-installer-inventory-file_platform-install-scenario[Editing the {PlatformName} installer inventory file].
+. Access your inventory file according to the procedure in link:h{BaseURL}/red_hat_ansible_automation_platform/2.3/html-single/red_hat_ansible_automation_platform_installation_guide/index#proc-editing-installer-inventory-file_platform-install-scenario[Editing the {PlatformName} installer inventory file].
```
Comment (ianf77) [inline downstream/modules/platform/ref-ldap-config-on-pah.adoc:29]: h before the attribute

File: downstream/modules/platform/ref-controller-use-oauth2-token-system.adoc [hunk 6]
```diff
@@ -28,6 +28,6 @@ These special OAuth 2 endpoints only support use of the `x-www-form-urlencoded`
 You can also request tokens using the `/api/o/token` endpoint by specifying `null` for the application type.
 ====
 
-Alternatively, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/automation_controller_user_guide/assembly-controller-applications#ref-controller-apps-add-tokens[Adding tokens] for users through the UI , and configuring the expiration of an access token and its associated refresh token (if applicable).
+Alternatively, see link:{BaseURL}/red_hat_ansible_automation_platform/2.4/html/automation_controller_user_guide/assembly-controller-applications#ref-controller-apps-add-tokens[Adding tokens] for users through the UI , and configuring the expiration of an access token and its associated refresh token (if applicable).
```
Comment (ianf77) [inline downstream/modules/platform/ref-controller-use-oauth2-token-system.adoc:31]: extra space between UI and ,

File: downstream/modules/platform/proc-controller-run-job-template.adoc [hunk 7]
```diff
@@ -14,15 +14,15 @@ image::controller-gs-job-templates-launch.png[Launch template]
...
+* link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/red_hat_ansible_automation_platform_installation_guide/index[Red Hat Ansible Automation Platform Installation Guide]
+* link:{BaseURL}/red_hat_ansible_automation_platform/2.4/html-single/automation_controller_administration_guide/index#doc-wrapper[_{ControllerAG}_]
 * link:http://docs.ansible.com/automation-controller/4.4/html/controllerapi/index.html#api-start[Automation Controller API Guide]
-* link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html-single/red_hat_ansible_automation_platform_release_notes/index#controller-440-intro[{ControllerNameStart} Release Notes]
-* link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4[Product Documentation for Red Hat Ansible Automation Platform]
+* link:{BaseURL}/red_hat_ansible_automation_platform/2.4/html-single/red_hat_ansible_automation_platform_release_notes/index#controller-440-intro[{ControllerNameStart} Release Notes]
+* link:{BaseURL}/red_hat_ansible_automation_platform/2.4[Product Documentation for Red Hat Ansible Automation Platform]
```
Comment (ianf77) [inline downstream/modules/platform/proc-controller-run-job-template.adoc:28]: should there be an /html-single/ in here?

File: downstream/modules/platform/proc-controller-create-container-group.adoc [hunk 8]
```diff
@@ -50,14 +50,14 @@ oc get secret $(echo ${SA_SECRET}) -o json | jq '.data.token' | xargs | base64 -
...
 
 To create a container group:
 
 .Procedure 
 
-. Use the {ControllerName} UI to create an link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html-single/automation_controller_user_guide/index#ref-controller-credential-openShift[OpenShift or Kubernetes API Bearer Token] credential to use with your container group.
-For more information, see https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html-single/automation_controller_user_guide/index#controller-getting-started-create-credential[Creating a credential] in the _{ControllerUG}_.
+. Use the {ControllerName} UI to create an link:{BaseURL}/red_hat_ansible_automation_platform/2.4/html-single/automation_controller_user_guide/index#ref-controller-credential-openShift[OpenShift or Kubernetes API Bearer Token] credential to use with your container group.
+For more information, see {BaseURL}/red_hat_ansible_automation_platform/2.4/html-single/automation_controller_user_guide/index#controller-getting-started-create-credential[Creating a credential] in the _{ControllerUG}_.
```
Comment (ianf77) [inline downstream/modules/platform/proc-controller-create-container-group.adoc:60]: missing link:

File: downstream/modules/platform/proc-completing-post-installation-tasks.adoc [hunk 9]
```diff
@@ -5,7 +5,7 @@
 [role="_abstract"]
 After you have completed the installation of {PlatformNameShort}, ensure that {HubName} and {ControllerName} deploy properly.
 
-// the section on adding a controller subscription duplicates content in https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/getting_started_with_automation_controller/controller-managing-subscriptions#doc-wrapper
+// the section on adding a controller subscription duplicates content in {BaseURL}/red_hat_ansible_automation_platform/2.4/html/getting_started_with_automation_controller/controller-managing-subscriptions#doc-wrapper
```
Comment (ianf77) [inline downstream/modules/platform/proc-completing-post-installation-tasks.adoc:8]: This doesn't look a proper link

**Review by ianf77 — COMMENTED** [review]
A few comments

File: downstream/modules/platform/proc-installing-the-aap-setup-bundle.adoc (hunk 4 above)
Comment (ariordan-redhat) [reply downstream/modules/platform/proc-installing-the-aap-setup-bundle.adoc:1]: No, it looks like a comment. It's an ancient link.

File: downstream/modules/platform/con-aap-installation-on-disconnected-rhel.adoc (hunk 1 above)
Comment (ariordan-redhat) [reply downstream/modules/platform/con-aap-installation-on-disconnected-rhel.adoc:27]: Created an attribute for Satellite.

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**